CHANGELOG.md

```diff
@@ -57,6 +58,10 @@ This project adheres to [Semantic Versioning](http://semver.org/).
    + Return new `PGRST303` error when jwt claims decoding fails
  - #3906, Return `PGRST125` and `PGRST126` errors instead of empty json - @taimoorzaeem
 
+### Deprecated
+
+- #3410, The config `server-timing-enabled` is deprecated. Use `metrics=timings` prefer header instead - @taimoorzaeem
```

Comment:
Now that `server-timing-enabled` has different functionality than `Prefer: metrics=timings`, I don't think we can deprecate it/remove it in the future. At least for now.